torch/nn/utils/fusion.py

```diff
@@ -16,6 +16,10 @@ def fuse_conv_bn_eval(conv, bn):
 def fuse_conv_bn_weights(conv_w, conv_b, bn_rm, bn_rv, bn_eps, bn_w, bn_b):
     if conv_b is None:
         conv_b = bn_rm.new_zeros(bn_rm.shape)
```

Comment:
Should it not be `conv_b = torch.zeros_like(bn_rm)` here as well?

Reply:
Will create another PR for that refactor.